lib/web/webidl/index.js

```diff
@@ -325,6 +325,10 @@ webidl.util.IsResizableArrayBuffer = function (V) {
   })
 }
 
+webidl.util.HasFlag = function (flags, attributes) {
+  return typeof flags === 'number' && (flags & attributes) === attributes
```

Comment:
do we really need to check if flags is a number? cant we expect that we are smart enough to pass numbers?